src/test/java/SalesforceModules/QuoteUtil.java

```diff
@@ -1253,4 +1253,138 @@ public static void deleteQuote_billing(String logStep) throws Exception {
         KeywordUtil.click(QuoteObject.deleteButtonClicked, logStep);
     }
 
-}
+    /**
+     * Create new quote combining all methods in billing section
```

Comment:
is it creating or editing the quote billing as from method name it looks like it is editing the quote

Reply:
done